Update Phone homes to include Pendo (#151)
hazelcast/management-center-docs: docs/modules/deploy-manage/pages/phone-homes.adoc

= Usage Analytics
:description: Hazelcast collects analytics data to learn about the usage of Hazelcast Management Center.
:page-aliases: ROOT:phone-home.adoc

{description}

Usage analytics of Management Center are collected in two ways:

* Management Center sends data to the Phone Home server at `\http://phonehome.hazelcast.com/pingMc` 30 minutes after it is started and once every 24 hours after that.

> * Pendo.io Javascript library is included in Management Center's web pages.  Pendo is used to generate reports on activities and analyze the performance of the Management Center's front-end.  You can find more information in Pendo's privacy policy at www.pendo.io/legal/privacy-policy/

Reviewer: ```suggestion
* The link:https://www.pendo.io/[Pendo] JavaScript library collects data to generate activity reports and to analyze the performance of the Management Center frontend. Pendo is included in all Management Center web pages. You can find information about Pendo's privacy policy at www.pendo.io/legal/privacy-policy/
```

* Screen height and width
* Window height and width

> == Disabling the collection of Usage Analytics

Reviewer: ```suggestion
== Disabling the Collection of Usage Analytics
```

> You can disable the Phone Homes and the Pendo's Javascript Library by setting the xref:system-properties.adoc#hazelcast-mc-phone-home-enabled[`hazelcast.mc.phone.home.enabled` property] to `false`

Reviewer: ```suggestion
You can disable data collection in phone homes and Pendo by setting the xref:system-properties.adoc#hazelcast-mc-phone-home-enabled[`hazelcast.mc.phone.home.enabled` property] to `false`
```
